src/components/list.js

```diff
@@ -1,11 +1,40 @@
 import React from 'react';
+import styled from 'styled-components';
+import { ListItem } from './ListItem';
 
 export const List = ({ items }) => {
-  console.log(items);
+  const [clicksCount, setClickCount] = React.useState(0);
+  const [clickedItemId, setClickedItemId] = React.useState('');
+
+  React.useEffect(() => {
```

Comment:
Šis ir viens variants, bet vai nebūtu vienkāršāk padot pogai onClick handleri, kas ir definēts šajā komponentē un izpildās uz katru kliku?